com.unity.netcode.gameobjects/Components/NetworkTransform.cs

```diff
@@ -605,7 +682,18 @@ internal bool ApplyTransformToNetworkState(ref NetworkTransformState networkStat
             return (isDirty, isPositionDirty, isRotationDirty, isScaleDirty);
         }
 
-        private void ApplyInterpolatedNetworkStateToTransform(NetworkTransformState networkState, Transform transformToUpdate)
+        /// <summary>
+        /// Applies the authoritative state to the local transform
+        /// </summary>
+        /// <remarks>
+        /// The serverTime is required for position and scale in order to prevent a single element of each 3 elements
+        /// from having too large of a delta time between the time it was last updated and the time any new update is
+        /// sent.
+        /// </remarks>
+        /// <param name="networkState">new state to apply</param>
```

Comment:
nit: these `param`s aren't adding much value - this is a private method so would recommend you just delete them